src/slack/views/postAnonymousReplySubmitted.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable camelcase */
+import { Middleware, ViewSubmitAction, SlackViewMiddlewareArgs } from '@slack/bolt';
+import { InputBlock, KnownBlock } from '@slack/types';
+import logger from '../../logger';
+import { app } from '../../app';
+import { env } from '../../env';
+
+export const postAnonymousReplySubmitted: Middleware<SlackViewMiddlewareArgs<ViewSubmitAction>> = async ({
+  ack,
+  body,
+  view,
+}) => {
+  ack();
+  try {
+    const { blocks, state, private_metadata } = view;
+    const replyBlockId = (blocks[1] as InputBlock).block_id;
+    const replyActionId = (blocks[1] as InputBlock).element.action_id;
+
+    const reply = state.values[replyBlockId][replyActionId].value;
+    const {
+      message_ts,
+      channel: { id },
+    } = JSON.parse(private_metadata);
+    const replyContext: KnownBlock[] = [
+      {
+        type: 'section',
+        text: {
+          type: 'mrkdwn',
+          text: reply,
+        },
+      },
+      {
+        type: 'context',
+        elements: [
+          {
+            type: 'mrkdwn',
+            text: 'This was sent *anonymously* and may not be the original poster',
```

Comment:
I just realized this can be used on any post, so there may not be an anonymous original poster. We could check to see if the original post came from the bot... if we want to do that, we'd have to add a new var to the slack setup stuff to track the bot ID and make it accessible to other parts of the app.

```suggestion
            text: 'This was sent *anonymously*',
```